File: app/src/main/java/org/techtown/find_gas_station/Retrofit/oilDetail/RESULT.java
package org.techtown.find_gas_station.Retrofit.oilDetail;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RESULT {

    @SerializedName("OIL")
    @Expose
    private OIL[] OIL;

    public OIL[] getOIL ()
    {
        return OIL;
    }

    public void setOIL (OIL[] OIL)
    {
        this.OIL = OIL;
    }

    @Override
    public String toString()
    {
        return "ClassPojo [OIL = "+OIL+"]";
    }
}

File: app/src/main/java/org/techtown/find_gas_station/data/remote/model/oilavg/Result.kt
package org.techtown.find_gas_station.data.remote.model.oilavg

import com.google.gson.annotations.SerializedName

data class OilAveragePriceInfoResult (
    @SerializedName("RESULT")
    val oilAveragePriceInfoResult : Result
)

data class Result(
    @SerializedName("OIL")
    val oilAveragePriceInfoDtoList: List<OilAveragePriceInfoDto>
)

data class OilAveragePriceInfoDto(
    @SerializedName("DATE")
    val date : String,

    @SerializedName("PRODCD")
    val oilKind : String,

    @SerializedName("PRICE")
    val oilPrice : String
)
